pylibftdi/driver.py

```diff
@@ -114,12 +122,11 @@ def _load_library(self, name, search_list=None):
                     lib = getattr(cdll, lib_path)
                     break
         if lib is None:
-            raise LibraryMissingError('{} library not found (search: {})'.format(
-                name, search_list))
+            raise LibraryMissingError(f'{name} library not found (search: {str(search_list)})')
```

Comment:
previous to introducing the `libraries` variable, this string included `search_list` which was the thing being iterated over - should probably still do that (maybe by reverting the introduction of `libraries` and just replacing your lines 104-109 with:

```python
if search_list is None:
    search_list = self._lib_search.get(name, [])
```

and that would be it...

Reply:
I implemented your suggestion (really a reversion to what you previously had anyway. :grimacing:)